src/hooks/productData/posthog_ai.tsx

```diff
@@ -302,101 +340,64 @@ export const posthog_ai = {
-        //         {
-        //             src: 'https://res.cloudinary.com/dmukukwp6/image/upload/v1730807086/workflows_90a59408f5.png',
-        //             alt: 'Workflows',
-        //             className: 'max-h-64',
-        //             // stylize: true,
-        //             // shadow: true,
-        //         },
-        //     ],
-        //     skills: [
-        //         {
-        //             name: 'Skill name',
-        //             description: 'Skill description',
-        //             sticker: <StickerPath className="size-6" />,
-        //             percent: 80,
-        //         },
-        //         {
-        //             name: 'Skill name',
-        //             description: 'Skill description',
-        //             sticker: <StickerPath className="size-6" />,
-        //             percent: 100,
-        //         },
-        //         {
-        //             name: 'Skill name',
-        //             description: 'Skill description',
-        //             sticker: <StickerPath className="size-6" />,
-        //             percent: 90,
-        //         },
-        //         {
-        //             name: 'Skill name',
-        //             description: 'Skill description',
-        //             sticker: <StickerPath className="size-6" />,
-        //             percent: 70,
-        //         },
-        //     ],
-        // },
         {
-            title: 'Error Tracking',
-            headline: 'Error Tracking',
+            title: 'Workflows',
+            headline: 'Workflows',
             layout: 'ai',
-            icon: <IconWarning className="size-5" />,
-            color: 'orange',
-            description:
-                'Find out what broke before your users tweet about it. PostHog AI helps identify error patterns and correlate signals across your product data.',
+            icon: <IconBolt className="size-5" />,
+            color: 'blue',
+            description: 'Create workflows that automate actions or send messages to your users.',
             images: [
                 {
-                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/v1730806655/error_tracking_4e0dff38af.png',
-                    alt: 'Error Tracking',
+                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/v1730807086/workflows_90a59408f5.png',
+                    alt: 'Workflows',
                     className: 'max-h-64',
                     // stylize: true,
                     // shadow: true,
                 },
             ],
             skills: [
                 {
-                    name: 'Stack trace decoder',
-                    description: 'Translates cryptic errors into "oh, THAT\'S the problem."',
+                    name: 'Create email templates',
+                    description:
+                        'Create email templates in your content library, including subject, body, html and liquid support',
```

Comment:
Would avoid using the same phrase in the title and description. Are we implying it's capable of writing forloops and inserting variables and all with natural language?